### PR TITLE
Add Mobile App / Project 3 Phase 1 report

### PR DESCRIPTION
**Pull request by Nitish145**

### CODE REVIEW

**Comment by Nitish145** [comment]
@satu0king @nitin10s please have a look!!

File: content/posts/mobile_app_phase_one_report.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+title: "Mobile App: Phase One Report"
+date: 2020-07-09T08:51:49+05:30
+draft: false
```
Comment (tachyons) [inline content/posts/mobile_app_phase_one_report.md:4]:
```suggestion
draft: false
author: Nitish Aggarwal ([Nitish145](https://github.com/Nitish145))
```

File: content/posts/mobile_app_phase_one_report.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+author: Nitish Aggarwal ([Nitish145](https://github.com/Nitish145))
+---
+
+The first evaluations happened from June 29th to July 3rd and Yay.. I Passed!!! This blog post comes as a summary to the amazing work done in the first phase of Google Summer of Code.
+
+<p align="center">
+	<img src="/img/first_eval_dashboard.png" alt = "First Evaluations Passed">
```
Comment (tachyons) [inline content/posts/mobile_app_phase_one_report.md:11]: Use markdown format for including images, html won't work https://deploy-preview-1--cvblog.netlify.app/posts/mobile_app_phase_one_report/

Comment (Nitish145) [reply content/posts/mobile_app_phase_one_report.md:11]: On it.. Just checked no images were being rendered :P

File: content/posts/mobile_app_phase_one_report.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+CircuitVerse now plans to expand.. GSoC 2020 basically aims at improving the Simulator and Web UI, structuring & refactoring the huge JS codebase, offline electron app, the mobile_app and implementing API.
+
+<br/>
+
+## The CircuitVerse API
+
+### Authentication & Users API
+
+For authenticating users, token auth is being used to `encode` user data and return the token for successful login/signup. The `private_key.pem` can also be fetched in `/private_key.pem` which could be used to `decode` the token to fetch user info on client side. Also added `/password/forgot` to send reset password instructions. Coming onto users, all users public info is made available at `/users` & logged in user at `/me`.
```
Comment (tachyons) [inline content/posts/mobile_app_phase_one_report.md:31]: public_key.pem, not private_key.pem

Comment (tachyons) [reply content/posts/mobile_app_phase_one_report.md:31]: Public key is not needed for decoding the token, it is only for validating the token signature

Comment (Nitish145) [reply content/posts/mobile_app_phase_one_report.md:31]: Oops! My bad, will updates these.

File: content/posts/mobile_app_phase_one_report.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+![First Evaluations Passed](/img/first_eval_dashboard.png)
+
+The mobile_app project was primarily focussed on two sub projects as follows:
+
+1. CircuitVerse API development.
+2. The Mobile App.
+
+## Overview
+
+Before the GSoC period CircuitVerse was primarily focussed for the Web. We have this very amazing [simulator](http://circuitverse.org/simulator), ability to create groups, beautiful circuits, assignments with grading obviously <3 and a lot more features. make sure to check them out on our [website](https://circuitverse.org/). Thanks to our amazing community, [@satu0king](https://github.com/satu0king), [@tachyons](http://github.com/tachyons/) & [@nitin10s](https://github.com/nitin10s).
```
Comment (nitin10s) [inline content/posts/mobile_app_phase_one_report.md:19]: Adding {target="_blank"} to all links would be better

**Comment by nitin10s** [comment]
screenshot
Why does the blog have such smaller width? Increasing the paragraph width slightly (refer Medium) would be definitely better.

**Comment by Nitish145** [comment]
@nitin10s i would agree with you on that.. @tachyons is this theme final or we may look onto some others too.. Also, i need to resize gifs included in the blog.. i tried some ways but were more or less not useful because the theme just adds a `width` of 100% to images..

Adding customised CSS for each of the blogs might not sound that good too.. Thoughts ?

**Comment by tachyons** [comment]
That is part of the theme, we can change the theme if needed.

**Comment by nitin10s** [comment]
The screenshots are covering all the screen space making the blog look odd. Reduce their height. Putting 2 screenshots in one row might work too.

**Pull request merged**